makefile.shade

```diff
@@ -102,6 +102,10 @@ var RUNTIME_TARGETS='${new List<RuntimeTarget>()}'
 var CAN_BUILD_ONECORE = '${Directory.Exists(VS_ONECORE_ROOT) && Directory.GetFiles(VS_ONECORE_ROOT, "vcruntime.lib", SearchOption.AllDirectories).Any() && Directory.Exists(WIN10_SDK_LIB) && Directory.GetFiles(WIN10_SDK_LIB, "onecore.lib", SearchOption.AllDirectories).Any()}'
 
 @{
+  E("CORECLR_RUNTIME_VERSION", "1.0.0-dev");
```

Comment:
This will execute unconditionally even on machines where `dev-install` has never run. Should instead change `test-package` target to create an alias for new runtime and set this environment variable there, just before calls to `DnxTest()`.